Gruntfile.js

```diff
@@ -25,7 +25,7 @@ module.exports = function(grunt) {
   // self-register with Raven via addPlugin
   function AddPluginBrowserifyTransformer() {
     return function(file) {
-      return through(function(buf, enc, next) {
+      return through(function(buf, _enc, next) {
```

Comment:
this function is wrong. It should be using _enc and it seems to want to add the require at the end of every chunk

Reply:
fixed in https://github.com/getsentry/raven-js/pull/1229/files